.buildkite/copy_files.py

```diff
@@ -79,9 +79,9 @@ def upload_paths(paths, resp, destination):
     for path in paths:
         fn = os.path.split(path)[-1]
         of["key"] = {
-            "wheels": f"latest/{sha}/{fn}",
+            "wheels": f"latest/{fn}",
```

Comment:
Great idea for providing "latest" wheels! Is it possible to have branch name in the path and upload latest wheels for each branch? That will make getting the latest release candidates easier too ...